1.16/s2i/bin/usage

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+DISTRO=`cat /etc/*-release | grep ^ID= | grep -Po '".*?"' | tr -d '"'`
+NAMESPACE=centos
+[[ $DISTRO =~ rhel* ]] && NAMESPACE=rhscl
+
+cat <<EOF
+This is a S2I ${IMAGE_DESCRIPTION} ${DISTRO} base image:
+To use it, install S2I: https://github.com/openshift/source-to-image
+
+Sample invocation:
+
+s2i build https://github.com/sclorg/nginx-container.git --context-dir=1.16/test/test-app/ ${NAMESPACE}/nginx-116-${DISTRO}7 nginx-sample-app
```

Comment:
`$NGINX_VERSION` can be used here, it will make it easier next time to convert to next nginx version.